pom.xml

```diff
@@ -316,6 +316,12 @@
       <version>6.0.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
While this marked optional, is it really required to compile here?  There isn't a single import on it.  So do we really need to state that?  IMO Aspectj is very old and interceptors replaced it a long time ago.  I rarely see code using it and manage some 2k repos in my day job and I can single digit count the ones that have aspectj.

Reply:
The new property `ComponentScan.Filter[]` in `@MapperScan` is reuse from Spring, and it support `FilterType.ASPECTJ` by using `org.springframework.core.type.filter.AspectJTypeFilter `, and this Class dependence on aspectjweaver, but  aspectjweaver is setted as optional dependency in spring-core, so I added this dependency, otherwise I can not use `org.springframework.core.type.filter.AspectJTypeFilter`.